### DISCUSSION TITLE
Is there any offline version of bootstrap tutorial

### DISCUSSION THREAD

**Discussion by thainee** [twbs/bootstrap#39607] [General]
![image](https://github.com/twbs/bootstrap/assets/87821262/03c05f1f-6077-4365-bf07-41f0db491a88)
Hello everyone, is there any link to download the offline version of bootstrap tutorial like this? And so that we can you live server to see the tutorial.

**Comment by mdo** [comment]
You're looking at React Bootstrap, which we don't maintain. Unsure if there's anything offline for it.
